packages/cli/oclif.manifest.json

```diff
@@ -171,6 +171,19 @@
           "required": false,
           "allowNo": false
         },
+        "custom-build": {
+          "name": "custom-build",
```

Comment:
I'm a little iffy on `custom-build` being the flag here because I could see that from a user's perspective that this would be the default. I can't think of something else, though!